src/Effects.js

```diff
@@ -281,9 +260,9 @@ module.exports = class Effects {
           const taco = this.tacos[i];
           p5.translate(taco.x, taco.y);
           p5.rotate(taco.rot);
-          p5.textAlign(p5.CENTER, p5.CENTER);
-          p5.textSize(this.size);
-          p5.text(String.fromCodePoint(55356, 57134), 0, 0);
+          p5.scale(2, 2);
+          drawTaco(p5._renderer.drawingContext);
+          p5.pop();
```

Comment:
FYI @epeach soon the taco will not be an emoji and we can re-enable it in visual tests!